src/components/Search.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import SearchBar from './SearchBar';
+import SearchResults from './SearchResults';
+import Autocomplete from './Autocomplete';
+import foodList from '../foodList';
+
+const Search = () => {
+  const [input, setInput] = useState('');
+  const [filteredList, setFilteredList] = useState(null);
+  const [result, setResult] = useState('');
+  const [pointer, setPointer] = useState(-1);
+
+  const updateInput = (input) => {
+    const inputString = input.trim().toLowerCase();
+    const filtered = foodList.filter((food) => {
+      return food.name.toLowerCase().includes(inputString);
+    });
+    setInput(input);
+    setFilteredList(inputString === '' ? null : filtered.slice(0, 5));
+    setPointer(-1);
+  };
+
+  const updateResults = (input) => {
+    const filtered = foodList.find((food) => {
+      return food.name.toLowerCase() === input.toLowerCase();
```

Comment:
This will repeatedly apply `String.prototype.toLowerCase()` to `input`, which can be calculated outside this arrow function. I strongly recommend creating a `const`, like you do in the `updateInput()` function above for optimal performance.